README.md

```diff
@@ -8,5 +8,9 @@ We are publicly sharing `infractions.yml`. `infractions.yml` is a data file of t
 
 The data file format is agnostic to any particular tool and is meant to be general enough to be adapted to any tool.
 
+## Adapters
+
+The `adapters` folder includes shell scripts to adapt the Airbnb infractions YAML file to third-party tools. Airbnb welcomes contributions for new tools and systems that promote belonging. An adapter for a third-party tool does not necesarily mean that Airbnb endorses the third-party tool.
```

Comment:
How do we review changes to adapters? Or put another way: how will we have the context to review changes to adaptors we don't endorse or know well?

Reply:
This is a good point. Let's remove the general concept of adapters until the need arises.

Reply:
https://github.com/airbnb/inclusion/pull/16/commits/37be8215801ef2d790a4113088416564a8120b28